games-roguelike/moria/moria-5.7.10.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils cmake-utils
```

Comment:
Do you need eutils here? I see you use newbin instead of make_wrapper anyway.

Reply:
No I don't. Forgot to remove. Thanks.